### ISSUE TITLE
Unable to download Dyndllod.  Installation failed.

### ISSUE THREAD

**Issue by MDBabb** [TitansBane/Elysium#149]
Receiving the following.  From reading previous comments it seems like I have to wait on a hotfix but I at least wanted to let someone know this was a new issue.

Thanks in advance for your time and the awesome Wabbajack integration!  you guys rock!  :-)

352.41 - Unable to download Elysium - Dyndolod 3D Output.7z (WabbajackCDNDownloader+State|https://authored-files.wabbajack.org/Elysium - Dyndolod 3D Output.7z_fad38c12-6737-4572-9381-53ad673f01eb)
352.41 - Unable to download Elysium - Dyndolod 3D Output (Creation Club).7z (WabbajackCDNDownloader+State|https://authored-files.wabbajack.org/Elysium - Dyndolod 3D Output (Creation Club).7z_1332a8a7-9f2f-4d1a-beb9-09e3b0256287)
352.41 - Unable to download Elysium - Dyndolod 3D Output - Performance Mode.7z (WabbajackCDNDownloader+State|https://authored-files.wabbajack.org/Elysium - Dyndolod 3D Output - Performance Mode.7z_3f224046-069f-4838-9f22-8f741c05c5b3)
352.41 - Unable to download Elysium - Dyndolod 3D Output - Performance Mode (Creation Club).7z (WabbajackCDNDownloader+State|https://authored-files.wabbajack.org/Elysium - Dyndolod 3D Output - Performance Mode (Creation Club).7z_ceffbdfc-1ad7-4d8b-8c24-1b4107ebb9f7)
352.41 - Unable to download Elysium - Texgen Output (Creation Club).7z (WabbajackCDNDownloader+State|https://authored-files.wabbajack.org/Elysium - Texgen Output (Creation Club).7z_50134245-d723-41f9-8831-16bd53181424)
352.41 - Unable to download Elysium - Texgen Output - Performance Mode (Creation Club).7z (WabbajackCDNDownloader+State|https://authored-files.wabbajack.org/Elysium - Texgen Output - Performance Mode (Creation Club).7z_21646802-47e6-4251-b9bd-154b6708db86)
352.41 - Cannot continue, was unable to download one or more archives

**Comment by lorenzini89** [comment]
Having the same issue. Guess well have to wait until this is fixed :)

**Comment by wsfield0** [comment]
Same issue here. Successfully downloaded everything yesterday but ran out of disc space during the extraction phase, then saw the modlist was under maintenance after making room. 

This morning I haven't been able to successfully download Dyndolod 3D output or Dyndolod 3D Output (Creation Club)

**Comment by RareformATB** [comment]
Hey guys, I don't know if you are still having issues, but I may have found a way to fix this issue. If you click the settings (the cog icon) on the main menu and turn on the network workaround option, it may be the help you need. I was stuck with this error as well, but enabling the network workaround got it past the instillation phase.

**Comment by lorenzini89** [comment]
> Hey guys, I don't know if you are still having issues, but I may have found a way to fix this issue. If you click the settings (the cog icon) on the main menu and turn on the network workaround option, it may be the help you need. I was stuck with this error as well, but enabling the network workaround got it past the instillation phase.

Thanks Rareform for stopping by and sharing your recommendation. Sadly its not working for me. Will keep trying tho.
